src/NuGet.Core/NuGet.Configuration/Settings/Items/CertificateItem.cs

```diff
@@ -72,8 +72,8 @@ public CertificateItem(string fingerprint, HashAlgorithmName hashAlgorithm, bool
             }
 
             AddAttribute(ConfigurationConstants.Fingerprint, fingerprint);
-            AddAttribute(ConfigurationConstants.HashAlgorithm, hashAlgorithm.ToString().ToUpper());
-            AddAttribute(ConfigurationConstants.AllowUntrustedRoot, allowUntrustedRoot.ToString(CultureInfo.CurrentCulture).ToLower());
+            AddAttribute(ConfigurationConstants.HashAlgorithm, hashAlgorithm.ToString().ToUpper(CultureInfo.InvariantCulture));
+            AddAttribute(ConfigurationConstants.AllowUntrustedRoot, allowUntrustedRoot.ToString(CultureInfo.CurrentCulture).ToLower(CultureInfo.CurrentCulture));
```

Comment:
Changing this to `ToLower(CultureInfo.InvariantCulture)` triggers https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1308

Reply:
We should disable that rule IMO, https://github.com/NuGet/Client.Engineering/issues/336. 

It's not helpful for us. We use a more limited charset and in a lot of cases, we have to compare to lower.

I don't think we should just blindly accept all validations.